lib/membrane/rtp/ssrc_router.ex

```diff
@@ -99,6 +100,9 @@ defmodule Membrane.RTP.SSRCRouter do
     register_new_inbound_track_event(pad, ctx)
     emit_new_inbound_track_event(ssrc, pad, ctx)
 
```

Comment:
In 98 we have `emit_packet_arrival_events` and it seems to be redundant as we always call `emit_packet_arrival_events` in handle_process.  Could we fix this?

Reply:
No, because this is not a fix, this is a regression. Packets are buffered if they arrive before their output pad is connected. This function call handles buffered packets, it's not redundant.

Reply:
You're right, thought that we always emit an event in handle_process but we do this only when buffer is not cached